can-view-target.js

```diff
@@ -83,56 +44,9 @@ var processNodes = function(nodes, paths, location, document){
  * @param {DocumentFragment} frag A document fragment to clone
  * @return {DocumentFragment} a new fragment that is a clone of the provided argument
  */
-var cloneNode = clonesWork ?
-	function(el){
-		return el.cloneNode(true);
-	} :
-	function(node){
-		var document = node.ownerDocument;
-		var copy;
-
-		if(node.nodeType === 1) {
-			if(node.namespaceURI !== 'http://www.w3.org/1999/xhtml' && namespacesWork && document.createElementNS) {
-				copy = document.createElementNS(node.namespaceURI, node.nodeName);
-			}
-			else {
-				copy = document.createElement(node.nodeName);
-			}
-		} else if(node.nodeType === 3){
-			copy = document.createTextNode(node.nodeValue);
-		} else if(node.nodeType === 8) {
-			copy = document.createComment(node.nodeValue);
-		} else if(node.nodeType === 11) {
-			copy = document.createDocumentFragment();
-		}
-
-		if(node.attributes) {
-			var attributes = node.attributes;
-			for (var i = 0; i < attributes.length; i++) {
-				var attribute = attributes[i];
-				if (attribute && attribute.specified) {
-					// If the attribute has a namespace set the namespace 
-					// otherwise it will be set to null
-					if (attribute.namespaceURI) {
-						copy.setAttributeNS(attribute.namespaceURI, attribute.nodeName || attribute.name, attribute.nodeValue || attribute.value);
-					} else {
-						copy.setAttribute(attribute.nodeName || attribute.name, attribute.nodeValue || attribute.value);
-					}
-				}
-			}
-		}
-
-		if(node && node.firstChild) {
-			var child = node.firstChild;
-
-			while(child) {
-				copy.appendChild( cloneNode(child) );
-				child = child.nextSibling;
-			}
-		}
-
-		return copy;
-	};
+var cloneNode = function(el){
```

Comment:
Can we remove this function and just call `el.cloneNode(true)` wherever this function is currently used?

Reply:
Yeah, but what if this https://github.com/canjs/can-view-target/blob/d20dab3619180104486a24793d47738b2d66952d/can-view-target.js#L289 is called elsewhere in another package?

Reply:
Done!

Reply:
Sorry, I didn't see that it was being exported. For that reason I think we should keep it in for backwards compatibility. Sorry for overlooking that.

Reply:
@matthewp is fixed!

Reply:
Thanks, looks good to me then.